### ISSUE TITLE
Wireframes for Member Dashboard

### ISSUE THREAD

**Issue by xprilion** [dsc-x/omg-badge#1]
Create wireframes for member dashboard, gallery and frames creator portal

**Comment by awantika10** [comment]
Hey! Can I work on this?

**Comment by Abhijay007** [comment]
@xprilion Sir can I work on this project, I will add react support to this project if you like. So can I contribute??

**Comment by xprilion** [comment]
@Abhijay007 Sorry, we're not currently looking to shift this project to React. You can take a look at the other issues or raise something of your own by testing the application! :)

**Comment by Abhijay007** [comment]
> @Abhijay007 Sorry, we're not currently looking to shift this project to React. You can take a look at the other issues or raise something of your own by testing the application! :)

ok got it 😄, I will try to work on some other issues.

**Comment by awantika10** [comment]
Made few wireframes of the Badge Generator with following features : 

- The Frame Generator consists of nav bar at the top with logo at the left , login functionality at right with undo and redo buttons.
-  The dashboard window is divided into two parts, first with a panel at left for changing the frames, styling the text, adding icons and uploading the user's image and second is the workspace for making the frames.
-  The workspace will also contain options for aligning the image and changing its orientation.
-  Download and Share buttons will be available below the frame. 

The theme and color schemes where taken according to [OMG Events Website](https://dscomg.com/)
![OMG1](https://user-images.githubusercontent.com/71998138/101980211-dbc15a00-3c89-11eb-86bb-ecbf74300850.PNG)
![OMG2](https://user-images.githubusercontent.com/71998138/101980212-dc59f080-3c89-11eb-943d-ea0533808ab7.PNG)
![OMG3](https://user-images.githubusercontent.com/71998138/101980209-da902d00-3c89-11eb-9840-8c1210aaa1e5.PNG)

**Comment by awantika10** [comment]
The following are the changes which were made according to @xprilion on the dashboard : 

- Increased the contrast between the panel and the dashboard for better visibility.
- Add Events feature rather than text for attendees, speaker and organizer of the event.
- Changed the color of the icons.

![omgomg](https://user-images.githubusercontent.com/71998138/102622422-00747080-4167-11eb-889b-cdcd3f937aa6.PNG)
![omgomg2](https://user-images.githubusercontent.com/71998138/102622423-010d0700-4167-11eb-9eb4-7ba02b33eae8.PNG)
![omgomg3](https://user-images.githubusercontent.com/71998138/102622415-feaaad00-4166-11eb-9579-1ba9ffce9e89.PNG)

**Comment by awantika10** [comment]
**New Feature : Website for OMG Badge Generator**
The following features has been added to the website : 

-  Login/SignIn : User can simply login and register themselves through the website by clicking on "Login".
-  For viewing the badge generator dashboard the user needs to hit "View Dashboard" button.
-  In order to view the previously made badges you need to click on "View Gallery" option.
-   The website will be available in Dark Mode as well.
![web](https://user-images.githubusercontent.com/71998138/102624479-15063800-416a-11eb-8e43-0ffb4427d262.PNG)
![DarkModeOMG](https://user-images.githubusercontent.com/71998138/102624402-f9029680-4169-11eb-9341-122bfa27747f.PNG)

**Comment by arnabsen1729** [comment]
@awantika10 Looks really good. Just a couple of edits from my side
 - SVGs above "Built for dev.." and "Designed to.. " not displayed in the dark mode. Same goes for the icon in the header/navbar.
 - Light mode to dark mode switcher button is missing in dark mode.
 - "View Gallery" text is not very clear in dark mode.
 
 Since you are following the Material UI format you can take some help from [here](https://material.io/design/color/dark-theme.html#ui-application)

**Comment by sairish2001** [comment]
@arnabsen1729 while implementation I will take care of it. Thanks

**Comment by awantika10** [comment]
Yeah. Me and @sairish2001 are working on the icons. We will take care of your suggestions.
Thanks :)

**Comment by awantika10** [comment]
**New Feature : Wireframe of the gallery**
-  Designed a simple wireframe of the gallery with home, dashboard and login option at the Nav bar.
-  Gallery will contain all the badges which the user has already made.
-  On hover, the user can edit, download and delete the frame.
-  With New Badge button, the user can make new logo from the dashboard.
![Icons (2)](https://user-images.githubusercontent.com/71998138/103151016-06d69e00-47a0-11eb-89b2-8e3eebe9c90a.png)

**Comment by awantika10** [comment]
**Update**

1. Created IWasAt logo using Illustrator.
2.  Added and Made a PR for Logo to the images folder.

**Couple of changes to be done for the UI** @sairish2001

1.  The Logo size should be increased according to the width of the nav bar.
![ytyt](https://user-images.githubusercontent.com/71998138/103453692-78fc3500-4d02-11eb-9957-79157525b7cd.PNG)

2.   The hexcode of the icons and text should be `#0094FF`, which is in accordance with the color of the logo.

3.   There are bugs related to the alignment of the buttons in the homepage and login page.
![ooooooooo](https://user-images.githubusercontent.com/71998138/103453789-88c84900-4d03-11eb-9f2e-71afc977ca37.PNG)
![uuuuuuuuuuuuu](https://user-images.githubusercontent.com/71998138/103453798-9aa9ec00-4d03-11eb-8caa-8487f8d27b17.PNG)

4.   The UI could be updated like this 
![Website Design (1)](https://user-images.githubusercontent.com/71998138/103453824-e8265900-4d03-11eb-8dd9-658fba379ec2.png)

**Comment by rajinderpalsingh2001** [comment]
Thanks for your valuable feedback. Will do in next Pull Request.
